src/components/Loading/Loading.tsx

```diff
@@ -12,10 +12,10 @@ const useStyles = makeStyles((_theme: Theme) => ({
     marginTop: '-28px',
   },
   fab: {
-    backgroundColor: '#fff',
+    backgroundColor: _theme.palette.common.white,
```

Comment:
Can you rename `_theme` to `theme`? Before it was an unused variable which eslint would complain about. So someone prefixed it with an `_` to bypass that lint rule. Sometimes that is useful if you don't need a specific argument of a function or item of an array but want to access something that comes after it.

Now that you are using it again, we don't need to prefix it.

Reply:
Will do. I didn't know why the underscore was there. I was just working with what was already provided.